app/portainer/components/datatables/registries-datatable/registriesDatatable.html

```diff
@@ -109,8 +109,8 @@
                 >
                   <pr-icon icon="'users'" feather="true"></pr-icon>Manage access
                 </button>
-                <be-feature-indicator feature="$ctrl.limitedFeature" ng-if="$ctrl.canBrowse(item)">
-                  <span class="text-muted space-left" style="padding-right: 5px"> <pr-icon icon="'search'" feather="true"></pr-icon> Browse </span>
+                <be-feature-indicator class="be-indicator" feature="$ctrl.limitedFeature" ng-if="$ctrl.canBrowse(item)">
```

Comment:
this component already have this class. see app/portainer/components/BEFeatureIndicator/BEFeatureIndicator.html

Reply:
If no such class, the indicator will not have an orange boarder.

> this component already have this class. see app/portainer/components/BEFeatureIndicator/BEFeatureIndicator.html

If no such class, the indicator will not have an orange boarder.

Reply:
so I think we will need to figure out why it's needed, because the underlying component already has this class

Reply:
https://github.com/portainer/portainer/blob/a247db7e9312372f34867cdc6d589e2f7173c8a0/app/portainer/components/BEFeatureIndicator/BEFeatureIndicator.html#L1

is missing the class. please apply it there and not here